.circleci/config.yml

```diff
@@ -40,111 +40,71 @@ jobs:
   tag-rc-candidate:
     description: Tag master as rc every 3 weeks
     docker:
-    - image: bylexus/apache-php55
-    - image: circleci/mysql:5.7
+    - image: cimg/base:2020.01
     steps:
-      - run:
-          name: Validate eligibility to tag rc
-          command: |
-            .circleci/scripts/rc-tag.sh
-      - when:
-          condition: $TAGTHISWEEK
-          steps:
-          - run:
-              name: Tag RC on Master
-              command: |
-                apt-get update
-                apt-get install -y git
-                apt-get install -y openssh-server
-          - swissknife/publish_github_release:
-              tag: $NEWTAGNAME
-          - notify-rc-tag:
-              tag: $NEWTAGNAME
+    - run:
+        name: Validate eligibility to tag rc
+        command: .circleci/scripts/rc-tag.sh
+    - when:
+        condition: $TAGTHISWEEK
+        steps:
+        - swissknife/publish_github_release:
+            tag: $NEWTAGNAME
+        - notify-rc-tag:
+            tag: $NEWTAGNAME
   unit-php55-mage18:
     docker:
-    - image: bylexus/apache-php55
+    - image: boltdev/php5.5-circle:0.0.4
     - image: circleci/mysql:5.7
     steps:
-    - run:
-        command: |
-          apt-get update
-          apt-get install -y git
-          apt-get install -y openssh-server
     - checkout
     - if-php-modified:
         steps:
         - run:
             name: PHP 5.5 Magento 1.8
-            command: |
-              mkdir ./artifacts
-              export TEST_ENV=php55-mage18
-              PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh nocov
-        - store_artifacts:
-            path: ./artifacts
+            command: MAGENTO_VERSION=magento-mirror-1.8.1.0 PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar tests/scripts/setup_and_test.sh nocov
         - notify-buildcop
   unit-php55-mage19:
     docker:
-    - image: bylexus/apache-php55
+    - image: boltdev/php5.5-circle:0.0.4
     - image: circleci/mysql:5.7
     steps:
-    - run:
-        command: |
-          apt-get update
-          apt-get install -y git
-          apt-get install -y openssh-server
     - checkout
     - if-php-modified:
         steps:
         - run:
             name: PHP 5.5 Magento 1.9
-            command: |
-              mkdir ./artifacts
-              export TEST_ENV=php55-mage19
-              PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.9.3.6 tests/scripts/ci.sh nocov
-        - store_artifacts:
-            path: ./artifacts
+            command: MAGENTO_VERSION=magento-mirror-1.9.3.6 PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar tests/scripts/setup_and_test.sh nocov
         - notify-buildcop
   unit-php56-mage18:
     docker:
-    - image: bylexus/apache-php56
+    - image: boltdev/php5.6-circle:0.0.1
```

Comment:
```suggestion
    - image: boltdev/php5.6-circle:0.0.4
```

Reply:
oh is only 5.6 at 0.0.1?

and 5.5 at 0.0.4?

Reply:
Yea cuz it took me a couple attempts to get 5.5 working. Once I got that working 5.6 worked in one try.